controllers/subscriptions.go

```diff
@@ -385,7 +385,32 @@ func GetStorageClusterSubscriptions() []*operatorv1alpha1.Subscription {
 		},
 	}
 
-	return []*operatorv1alpha1.Subscription{noobaaSubscription, ocsSubscription, csiAddonsSubscription}
+	rookSubscription := &operatorv1alpha1.Subscription{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      RookSubscriptionName,
+			Namespace: OperatorNamespace,
+		},
+		Spec: &operatorv1alpha1.SubscriptionSpec{
+			CatalogSource:          RookSubscriptionCatalogSource,
+			CatalogSourceNamespace: RookSubscriptionCatalogSourceNamespace,
+			Package:                RookSubscriptionPackage,
+			Channel:                RookSubscriptionChannel,
+			StartingCSV:            RookSubscriptionStartingCSV,
+			InstallPlanApproval:    operatorv1alpha1.ApprovalAutomatic,
+			Config: &operatorv1alpha1.SubscriptionConfig{
+				Tolerations: []corev1.Toleration{
+					{
+						Key:      "node.ocs.openshift.io/storage",
+						Operator: "Equal",
+						Value:    "true",
+						Effect:   "NoSchedule",
```

Comment:
It is better to have it as part of the CSV itself.

Reply:
cc @subhamkrai please take care of this in the rook PR